sechub-cli/src/daimler.com/sechub/cli/help.go

```diff
@@ -25,8 +25,9 @@ func PrintUsage() {
 	action += "   " + ActionExecuteGetReport + " - fetch report as json (result will only exist when job is done)\n"
 	action += "   " + ActionExecuteGetFalsePositives + " - fetch the project's false-positives list as json\n"
 	action += "   " + ActionExecuteMarkFalsePositives + " - add from a json file to project's false-positives list\n"
-	action += "   " + ActionExecuteInteractiveMarkFalsePositives + " - interactively ask for false-positives depending on a json report\n"
-	action += "   " + ActionExecuteUnmarkFalsePositives + " - remove from project's false-positives list defined in json file\n"
+	action += "   " + ActionExecuteUnmarkFalsePositives + " - remove items from project's false-positives list as defined in json file\n"
```

Comment:
A small helper method for the formatting would be a good idea here.

Reply:
I think a helper method is not necessary - reason: this is go and the kind of string concatenation is okay here.
Also help method call would be like:

callme(actionRef, "       ",ActionExecuteInteractiveMarkFalsePositives, "interactively ask for false-positives depending on a json report"

the "-" and "\n" parts would be less, but it would also be less readable.
So I think we keep this as is.